WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderViewModel.kt

```diff
@@ -220,6 +222,29 @@ class ReaderViewModel @Inject constructor(
     @Subscribe(threadMode = MAIN)
     fun onTagsUpdated(event: ReaderEvents.FollowedTagsFetched) {
         loadTabs()
+        // Determine if analytics should be bumped either due to tags changed or time elapsed since last bump
+        val now = DateProvider().getCurrentDate().time
+        val shouldBumpAnalytics = event.didChange()
+                || ( now - appPrefsWrapper.readerAnalyticsCountTagsTimestamp > ONE_HOUR_MILLIS)
+
+        if (shouldBumpAnalytics) {
+            readerTracker.trackFollowedTagsCount(event.totalTags)
+            appPrefsWrapper.readerAnalyticsCountTagsTimestamp = DateProvider().getCurrentDate().time
```

Comment:
💡 Nitpick/optional: maybe we can use `now` defined in line `226` instead of getting current date again?